samples/matter/lock/README.rst

```diff
@@ -302,15 +306,15 @@ You can enable the :ref:`matter_lock_sample_ble_nus` feature by setting the :kco
    Matter commissioning, DFU, and NUS over Bluetooth LE must be run separately.
 
 The door lock's Bluetooth LE service extension with NUS requires a secure connection with a smartphone, which is established using a security PIN code.
-The PIN code is different depending on the build type:
+The PIN code is different depending on the :ref:`configuration <matter_lock_sample_custom_configs>` the sample was built with:
 
-* In the ``debug`` build type, the secure PIN code is generated randomly and printed in the log console in the following way:
+* In the debug configuration, the secure PIN code is generated randomly and printed in the log console in the following way:
```

Comment:
Shouldn't it be done like for example in the template readme file: https://github.com/nrfconnect/sdk-nrf/pull/15942/files#diff-0077c6a5bc707dcb25f2cc3cd826ee69d719cf2d0bd135729e4f52d7c94437d6R120
To highlight the configuration and link to the table?

Reply:
I thought it would be enough to link the configuration once on the introducing line (Line 309).